File: java/trunk/TetramBDEJB/ejbModule/org/tetram/bdtheque/model/entity/ColoristeSerie.java
package org.tetram.bdtheque.model.entity;

import javax.persistence.DiscriminatorValue;
import javax.persistence.Entity;

@Entity
@DiscriminatorValue("2")
public class ColoristeSerie extends AuteurSerie {

  /**
	 * 
	 */
  private static final long serialVersionUID = 1L;

}

File: bdtheque/php/trunk/src/BDTheque/DataModelBundle/Entity/ColoristeSerie.php
<?php

namespace BDTheque\DataModelBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * BDTheque\DataModelBundle\Entity\ColoristeSerie
 * 
 * @ORM\Entity
 */
class ColoristeSerie extends AuteurSerie {

    public function setSerie(Serie $serie) {
        $serie->addColoriste($this);
        return parent::setSerie($serie);
    }

}

?>